.circleci/config.yml

```diff
@@ -201,9 +202,17 @@ jobs:
           command: |
             docker run \
               --env-file docker-cache/.env \
-              --name reactionapp_next_starterkit_ssr \
+              --name reactionapp_next_starterkit_int \
               "$DOCKER_REPOSITORY:$CIRCLE_SHA1" \
-              yarn run test:integration  
+              yarn run test:integration
+      - run:
+          name: Install Broken Link Checker
+          command: |
+            sudo npm i -g broken-link-checker
```

Comment:
Would this dep not be installed during the build step? If its needed should it be `yarn install`?

Reply:
It is not installed during this step. 

I initially vouched for `npm` rather than `yarn` for the global option because the step that actually calls `blc` cannot find the command due to the way how `run` blocks are isolated which leads to example output: 

```
/bin/bash: blc: command not found
Exited with code 127
```

However, if done by `yarn add` one has to call the executable through `./node_modules/.bin/blc` I changed it to that way.